pycm/pycm_param.py

```diff
@@ -23,6 +23,9 @@
 MATRIX_FORMAT_ERROR = "Input confusion matrix format error"
 MAPPING_FORMAT_ERROR = "Mapping format error"
 MAPPING_CLASS_NAME_ERROR = "Mapping class names error"
+INVALID_PLOT_LIBRARY_ERROR = "{} is not a valid plotting library"
+SEABORN_PLOT_LIBRARY_ERROR = "Importing seaborn module error"
+MATPLOTLIB_PLOT_LIBRARY_ERROR = "Importing matplotlib module error"
```

Comment:
I think this message is better :
`Error in importing matplotlib module. Please install it : pip install matplotlib`
@alirezazolanvari What's your idea ?